lib/pltr/v2/reducers/tours.js

```diff
@@ -8,56 +9,64 @@ export default function tours(state = INITIAL_STATE, action) {
     case SET_TOUR_FEATURE:
       const newState = Object.assign({}, INITIAL_STATE)
       newState.feature = action.payload //an object with feature: 'string', featureId: int
+      newState.steps = ACTS_TOUR_STEPS
       newState.showTour = true
       return { ...state, ...newState }
     //main tour callback
     case TOUR_NEXT: {
       let transitionSteps = []
       let endStep
-      state.b2bTransition = false
+      const newState = Object.assign({}, state)
+
+      //ensure the tour runs at the next step
+      newState.loading = false
+      newState.run = true
+
       //switch based on which tour is running -> to determine when the
       //tour callback should delay until next step's component is mounted
-      switch (state.feature.name) {
+      switch (newState.feature.name) {
         case 'acts':
-          transitionSteps = [0, 4, 5, 6]
-          endStep = 9
+          transitionSteps = [0,4,5,6,7]
+          endStep = 8
           break
         default:
-          state.transitioning = false
-      }
-      //delay the next step until component associated with the step is mounted
-      if (transitionSteps.includes(state.stepIndex)) {
-        state.transitioning = true
-        //need to anticipate back-to-back delays
-        if (transitionSteps.includes(state.stepIndex + 1)) {
-          state.b2bTransition = true
-        }
-      } else {
-        state.transitioning = false
+          newState.transitioning = false
       }
+
       //increment or decrement stepIndex
       if (action.payload === 'prev') {
-        state.stepIndex = state.stepIndex - 1
+        newState.stepIndex = newState.stepIndex - 1
       } else {
-        state.stepIndex = state.stepIndex + 1
+        newState.stepIndex = newState.stepIndex + 1
       }
-      //ensure the tour runs at the next step
-      state.loading = false
-      state.run = true
+
+      newState.transitioning = false
+
+      //delay the next step until component associated with the step is mounted
+      if (transitionSteps.includes(newState.stepIndex)) {
+        newState.transitioning = true
+      } 
+
       //stop the tour at the appropriate endStep,
       //store that they have completed this tour,
       //clear the feature so the tour doesn't run again unless manually started
-      if (state.stepIndex === endStep) {
-        state.toursTaken[`${state.feature.name}`] = true
-        state.feature = {}
-        state.run = false
+      if (newState.stepIndex === endStep) {
+        newState.toursTaken[`${newState.feature.name}`] = true
+        newState.feature = {}
+        newState.run = false
       }
-      state.showTour = false
-      return { ...state, state }
+      
+      return { ...state, ...newState }
     }
+    case TOUR_TRANSITION: 
+      const nextState = Object.assign({}, state)
+      nextState.run = false
+      nextState.loading = true
+      nextState.transitioning = false
+      return {...state, ...nextState}
```

Comment:
You don't need to spread `state` here either.
We generally do this when we want a new version of state with just a few properties changed.